helm/dagster/schema/schema/charts/dagster/subschema/daemon.py

```diff
@@ -82,6 +87,7 @@ class Daemon(BaseModel):
     annotations: kubernetes.Annotations
     runMonitoring: Dict[str, Any]
     runRetries: Dict[str, Any]
+    sensors: Optional[Sensors]
```

Comment:
Technically it's not optional, as it has a default specified in `values.yaml`. 

```suggestion
    sensors: Sensors
```

Reply:
Well, I want to avoid the backcompat breakage (old user code against new helm upgrade), so should I remove the settings in values.yaml?